documentation/docs/tutorials/value-transactions/01-introduction.md

```diff
@@ -0,0 +1,50 @@
+---
+description: "Get acquainted with the structure and functionality of transactions in IOTA's Stardust protocol in the
+Shimmer network using iota.js primitives to issue value transactions."
+image: /img/client_banner.png
+keywords:
+
+- tutorial
+- UTXO
+- transactions
+- shimmer
+- ledger
+- outputs
+- inputs
+
+---
+
+# Stardust for iota.js: Value Transactions
+
+This tutorial will guide you as you get acquainted with the structure and functionality of transactions in
+IOTA's [Stardust protocol](https://wiki.iota.org/shimmer/introduction/welcome) in
+the [Shimmer network](https://shimmer.network) using [iota.js](https://github.com/iotaledger/iota.js) primitives to
+issue value transactions.
+
+## Introduction
+
+Starting from [Chrysalis](https://wiki.iota.org/introduction/welcome), IOTA is
+a [UTXO-based (Unspent Transaction Output)](https://wiki.iota.org/introduction/reference/details#unspent-transaction-output-utxo)
+Ledger. Each UTXO, also known as  **output**, has an associated number of tokens that determines its value.
+Thus, the permanent data on the ledger is composed of a set of records (*outputs*) that can be unlocked by the owner of
+its associated address, i.e., the one who knows the address' private key.
+
+There are different [output types](https://wiki.iota.org/shimmer/learn/outputs). This tutorial series will only focus on
+traditional value outputs known as [`IBasicOutput`](./../../references/client/interfaces/IBasicOutput.md). Outputs are
```

Comment:
I don't think that "traditional" is a right word here.

Reply:
```suggestion
value outputs known as [`IBasicOutput`](./../../references/client/interfaces/IBasicOutput.md). Outputs are
```